nntrainer/utils/meson.build

```diff
@@ -1,6 +1,7 @@
 util_sources = [
   'util_func.cpp',
   'profiler.cpp',
+  'tracer.cpp',
```

Comment:
add tracer.cpp only when enable-trace is true. That helps build-modularizing.

Reply:
Thanks. It modifed.